Makefile

```diff
@@ -109,6 +109,10 @@ linkcheck-external:
 	docker kill server
 	docker kill linkchecker
 
+grab-main-site-header-footer:
+	curl -s https://www.giantswarm.io/why-giant-swarm | sed -n '/^<!-- BEGIN SITE_HEADER -->/,/^<!-- END SITE_HEADER -->/p;/^<!-- END SITE_HEADER -->/q' | sed '1d;$d' > src/layouts/partials/gs_header.html
+	curl -s https://www.giantswarm.io/why-giant-swarm | sed -n '/^<!-- BEGIN SITE_FOOTER -->/,/^<!-- END SITE_FOOTER -->/p;/^<!-- END SITE_FOOTER -->/q' | sed '1d;$d' > src/layouts/partials/gs_footer.html
```

Comment:
Can you add a section about to use this to the `README` or `CONTRIBUTING` file?

Also can we add a little safety here? I'm thinking about cases where curl fails, or returns some unexpected output.

Maybe just a check that the file size is not below a certain value? For example with `[ "$(wc -c < src/layouts/partials/gs_header.html)" -ge 2000 ]` (2000 is arbitrary)
Maybe just a check

Reply:
I don't wanna over think this part right now, also don't understand the check yet. I think maybe we can take this in another PR that tries to automate this further. For now it's like a little hint to the future me or whoever takes it on next, as to what gs_header and gs_footer really contain.